content/pages/new-committers-guide.md

```diff
@@ -1,6 +1,6 @@
 Title: Guide for new project committers
 
-Here's how to set up the technical and social environment that will support your work as a committer to an Apache project. Some projects have more specific guidelines which the project website or the PMC provide for you. We also have a thorough [Committers' FAQs](commiters.html) for both new and experienced comnmitters.
+Here's how to set up the technical and social environment that will support your work as a committer to an Apache project. Some projects have more specific guidelines which the project website or the PMC provide for you. We also have a thorough [Committers' FAQs](commiters.html) for both new and experienced committers.
 
```

Comment:
There is actually another typo on the same line - commiters.html.
If you can push a fix to your PR for that, should be good for merge

Reply:
Ok, fix the broken link